apps/common/lib/lexical/ast/analysis/import.ex

```diff
@@ -35,6 +35,11 @@ defmodule Lexical.Ast.Analysis.Import do
                   {{:__block__, _, [function_name]}, {:__block__, _, [arity]}} ->
                     {function_name, arity}
                 end)
+
+              _ ->
```

Comment:
Should we consider the situation with sigils?

https://github.com/elixir-lsp/elixir_sense/blob/c3771bc69ec7a907b693f6f11983e38da21848d4/test/support/fixtures/metadata_builder/import/only_sigils.ex#L3

https://github.com/lexical-lsp/lexical/pull/535/commits/026a4d709887b27176abbfef31cb9f7f3b5b3d66

Reply:
yes, that one slipped by me